src/site/_includes/components/component-preview/macro.njk

```diff
@@ -18,8 +18,8 @@
     </a>
   </div>
   <div class="guidance-component-preview__preview-wrapper">
-    <div class="guidance-component-preview__preview">
-      <div class="guidance-component-preview__preview-component">
+    <div class="guidance-component-preview__preview {% if params.iframe %} js-guidance-component-preview-iframe guidance-component-preview__preview--iframe{% endif %}" data-example-name="{{ params.exampleName }}" data-iframe-id="{{ idPrefix }}--iframe">
```

Comment:
The iframe needs to replace `guidance-component-preview__preview` element I found that if it is a child it does not respond too well.  I think it maybe because this class has the `resize: both` rule on it